config/config_test.go

```diff
@@ -124,17 +123,13 @@ func Test_Parse_Checks(t *testing.T) {
 	}
 	for _, tt := range tests {
 		tt := tt // Re-initializing variable so it is not changed while executing the closure below
```

Comment:
optional: now that the import cycle is broken, lines 15-17 (wont let me pick them) can be changed just to `package config`?

But only if you feel like it, not important.